BedrockCommand.cpp

```diff
@@ -210,6 +213,7 @@ void BedrockCommand::finalizeTimingInfo() {
           << commitSyncTotal/1000 << ", "
```

Comment:
@tylerkaraszewski do we need to add a different timing for the "commit"? something like commitBlockingTotal? or that is not needed?

Reply:
Spoke with Aldo 1:1 that we're interested in tracking the time in which the command ran

Reply:
We already track commit time, it's not different for blocking/non-blocking transactions.